basic-javascript/project-1.spec.js

```diff
@@ -1,6 +1,10 @@
 const helpers = require('./project-1')
 
-// start testing!
+//tests
+/**
```

Comment:
very nice in going further and doing param, returns

Reply:
TY!